src/qtgui/plotter.cpp

```diff
@@ -1270,7 +1270,6 @@ void CPlotter::drawOverlay()
     QFontMetrics    metrics(m_Font);
     QPainter        painter(&m_OverlayPixmap);
 
-    painter.initFrom(this);
```

Comment:
https://doc.qt.io/qt-5/qpainter-obsolete.html#initFrom

This is obsolete and no replacement is suggested. The plotter appears to work fine without it, so I just removed it.